src/sentry/snuba/discover.py

```diff
@@ -607,7 +607,7 @@ def top_events_timeseries(
                         if zerofill_results
                         else item["data"]
                     ),
-                    "groupby": item.get("groupby", []),
+                    "groupby": item.get("groupby", None),
```

Comment:
This shouldn't cause any issues elsewhere right? Like, we're not expecting an iterable wherever this is consumed downstream?

Reply:
Not consumed downstream yet :+1: and this matches the spec better than before :+1: